src/utilities/number/index.ts

```diff
@@ -81,7 +81,7 @@ export const removeFormat = (formattedValue: string | undefined): string | undef
  */
 export const toUSFormat = (value: string | undefined): string | undefined => {
   if (withCommaDecimalSeparator()) {
-    return value?.split('.').join('-').split(',').join('.').split('-').join(',');
+    return value?.split('.').join('#').split(',').join('.').split('#').join(',');
```

Comment:
are test passing after this?

Reply:
Yes, tests are passing -- using `-` as the auxiliary character would break logic when number are negative

Reply:
Added more tests [f834398](https://github.com/Voltz-Protocol/voltz-ui/pull/570/commits/f834398de9a6df0179b6a7fe7ce15c518c7adaae) to catch this scenarios